src/main/io/serial_msp.c

```diff
@@ -1868,6 +1876,16 @@ static bool processInCommand(void)
         masterConfig.baro_hardware = read8();
         masterConfig.mag_hardware = read8();
         break;
+        
+    case MSP_SET_NAME:
+        memset(masterConfig.name, '\0', MAX_NAME_LENGTH);
```

Comment:
memset should set all the chars in the array to 0 (i.e. ARRAYLEN() - as it is MAX_NAME_LENGTH + 1). You can simply use the value 0 here, no need for '\0'.